app/controllers/concerns/referer.rb

```diff
@@ -16,7 +16,7 @@ def from_internet?
   end
 
   def request_host_include?(hosts)
-    hosts.any? { |host| Addressable::URI.parse(request.referer).host.include? host } rescue false
```

Comment:
I think this was failing before this PR, right?

Are there any tests for this?